src/fdc3/dotnet/DesktopAgent/src/MorganStanley.ComposeUI.DesktopAgent/DependencyInjection/ServiceCollectionExtensions.cs

```diff
@@ -34,6 +35,7 @@ public static class ServiceCollectionExtensions
             builderAction(builder);
         }
 
+        serviceCollection.AddSingleton<IMessagingService>(provider => provider.GetRequiredService<IMessageRouter>());
```

Comment:
As discussed offline adding this registration together with the IMessageRouter registration is worth considering

Reply:
Done.